repository/src/main/java/org/alfresco/repo/security/authentication/identityservice/IdentityServiceConfig.java

```diff
@@ -306,4 +313,19 @@ public void setAdminConsoleRedirectPath(String adminConsoleRedirectPath)
     {
         this.adminConsoleRedirectPath = adminConsoleRedirectPath;
     }
+
+    public Set<SignatureAlgorithm> getSignatureAlgorithms()
+    {
+
+        return Stream.of(signatureAlgorithms.split(","))
+            .map(String::trim)
+            .map(SignatureAlgorithm::from)
+            .filter(Objects::nonNull)

```

Comment:
It would be good to log a warning message if we are not able to create a `SignatureAlgorithm` for a given `String`. It might help in identifying the configuration issues.

Reply:
Done.